### PR TITLE
CSS Grid Wrapper + Google Login Functionality

### PR DESCRIPTION
**Pull request by justinjeon**
Added the /login route. Created a reusable component for the CSS styling of the app. Also added a Google authentication button.

### CODE REVIEW

File: frontend/src/index.html [hunk 1]
```diff
@@ -5,6 +5,9 @@
     <meta name="viewport" content="width=device-width">
     <link href="https://fonts.googleapis.com/css?family=Karla" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Inconsolata" rel="stylesheet">
+    <meta name="google-signin-scope" content="profile email">
+    <meta name="google-signin-client_id" content="YOUR_CLIENT_ID.apps.googleusercontent.com">
```
Comment (ramsgoli) [inline frontend/src/index.html]: Do we need this line? And the one above

File: frontend/src/containers/Login/index.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      email: '',
+      password: '',
+    }
+  }
+
+  responseGoogle = response => {
+    console.log(response)
+  }
+
+  validateForm() {
```
Comment (ramsgoli) [inline frontend/src/containers/Login/index.js]: Is this function used?

File: frontend/src/containers/Login/index.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import styled from 'styled-components'
+import { GoogleLogin } from 'react-google-login'
+import Wrapper from '../../components/Wrapper'
+// import ReactDOM from 'react-dom'
```
Comment (ramsgoli) [inline frontend/src/containers/Login/index.js]: Better to remove commented code if its not needed

File: frontend/src/index.html (hunk 1 above)
Comment (justinjeon) [reply frontend/src/index.html]: I commented them out and the google button still works, so I will delete them

**Pull request merged**